src/__tests__/__snapshots__/theme.js.snap

```diff
@@ -12,66 +12,67 @@ Object {
   "colors": Object {
     "black": "#000",
     "blue": "#007aff",
-    "blue0": "#cce4ff",
-    "blue1": "#99caff",
-    "blue2": "#66afff",
-    "blue3": "#3395ff",
+    "blue0": "#cdf",
+    "blue1": "#007aff",
+    "blue2": "#049",
     "blues": Array [
-      "#cce4ff",
-      "#99caff",
-      "#66afff",
-      "#3395ff",
+      "#cdf",
+      "#007aff",
+      "#049",
     ],
-    "gray": "#596b7d",
-    "gray0": "#dee1e5",
-    "gray1": "#bdc4cb",
-    "gray2": "#9ba6b1",
-    "gray3": "#7a8997",
+    "darkBlue": "#049",
```

Comment:
Are the dark colors not defined as a numbered color? Would be good to have a scale for the entire range of each hue, then add shorthands for `darkBlue` and `lightBlue` IMO

Reply:
Disregard this. Too early and commenting on snapshots 😬